packages/web/src/lib/web-mode.ts

```diff
@@ -0,0 +1,133 @@
+import algosdk, { ConfirmedTxInfo, SuggestedParams, Transaction } from "algosdk";
+
+import { AlgoSigner, AlgoSignerSendTx, AlgoSignerSignedTx, AlgoSignerToBeSignedTx, ExecParams, TxParams } from "../types";
+import { mkTransaction } from "./txn";
+
+const CONFIRMED_ROUND = "confirmed-round";
+const LAST_ROUND = "last-round";
+
+export class WebMode {
+  algoSigner: AlgoSigner;
```

Comment:
maybe this should be private ?

Reply:
i think this is good, user can use `Algosigner` anyway because user has passed it to this constructor